src/workloads/scale/LargeScaleSerial.yml

```diff
@@ -1,7 +1,7 @@
 SchemaVersion: 2018-07-01
 Owner: Storage Engines
 Description: |
-  TODO: TIG-3319
+  An example large scale workload file created to prove that such a thing was possible.
```

Comment:
I was going to ask for more info on this rather than a one-liner, but I see we didn't create this, but we are marked as the owner...?  It seems to be related to LargeScaleParallel.yml (which we are not marked as the owner).

Reply:
I was wondering this too, I don't think we are the original author so instead of updating the description I'll change the ownership to whichever team we think owns it?